src/test/java/com/exasol/RestrictedAccessIT.java

```diff
@@ -27,23 +26,6 @@ void testAccessRoleProtectedTableWithoutRole() {
         assertRlsQueryWithUser("SELECT C1 FROM " + virtualSchema.getName() + ".T", user, table("VARCHAR").matches());
     }
 
-    @Test
-    void testAccessRoleProtectedTableWhenUserMappingIsMissing() {
-        assumeExasolSevenOneOrLater();
-        final Schema schema = createSchema("SCHEMA_FOR_ACCESS_WITHOUT_ROLE");
-        schema.createTable("T", "C1", "VARCHAR(20)", ROW_ROLES_COLUMN, ROLE_MASK_TYPE) //
-                .insert("NOT ACESSIBLE", bitsToLong(0));
-        final VirtualSchema virtualSchema = createVirtualSchema(schema);
-        final User user = createUserWithVirtualSchemaAccess("USER_FOR_ACCESS_WITHOUT_ROLE", virtualSchema);
-        assertRlsQueryWithUser("SELECT C1 FROM " + virtualSchema.getName() + ".T", user, table("VARCHAR").matches());
-    }
-
-    private void assumeExasolSevenOneOrLater() {
-        final ExasolDockerImageReference imageReference = EXASOL.getDockerImageReference();
-        assumeTrue((imageReference.getMajor() > 7)
-                || ((imageReference.getMajor() == 7) && (imageReference.getMinor() > 0)));
-    }
-
```

Comment:
is it no longer needed?

Reply:
On second thought I added a test that with the new way of generating the push-down SQL checks that the generated query with the `JOIN` throws a proper compilation error if the mapping is missing.